spec/operators/filter-spec.ts

```diff
@@ -1,6 +1,12 @@
 import {expect} from 'chai';
 import * as Rx from '../../dist/cjs/Rx';
-declare const {hot, cold, asDiagram, expectObservable, expectSubscriptions};
+import marbleTestingSignature = require('../helpers/marble-testing'); // tslint:disable-line:no-require-imports
```

Comment:
I haven't tried it but tempted to leave as-is to give clarity this import is unique kind and not being used in general.